packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -369,22 +371,24 @@ function commitProfilerPassiveEffect(
         // It does not get reset until the start of the next commit phase.
         const commitTime = getCommitTime();
 
+        let phase = finishedWork.alternate === null ? 'mount' : 'update';
+        if (enableProfilerNestedUpdatePhase) {
+          if (isCurrentUpdateNested()) {
+            phase = 'nested-update';
```

Comment:
What are your thoughts on `update-nested` instead? That seems to read more like "Update (nested)" to me.

Reply:
Hm. I think it reads a bit more awkward. When we talk about these things, we call them "nested updates".